docs/06_error_handling.md

```
@@ -1,112 +1,165 @@
+so it produces a value, and a `finally` block may follow the arms.
 
-In `async`-`await` functions, the type of exceptions that can be throw 
+A `try` with no catch arms is rejected:
 
-```ts
-class FetchError extends Error {
-    constructor(url: string) {
-        super(`failed to fetch ${url}`)
-    }
-}
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)